Formula/openvpn.rb

```diff
@@ -72,7 +73,7 @@ def caveats
 
     s += <<-EOS.undent
       For OpenVPN to work as a server, you will need to create configuration file
-      in #{etc}/openvpn, samples can be found in #{share}/doc/openvpn
+      in #{etc}/openvpn, samples can be found in #{doc}/samples
```

Comment:
Yeah, probably to be fair. `etc.install` shouldn't overwrite someone's existing `.conf`.